src/HealthChecks.Publisher.Datadog/HealthChecks.Publisher.Datadog.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetStandardTargetVersion)</TargetFramework>
+    <PackageLicenseUrl>$(PackageLicenseUrl)</PackageLicenseUrl>
+    <PackageProjectUrl>$(PackageProjectUrl)</PackageProjectUrl>
+    <PackageTags>HealthCheck;Publisher;Datadog</PackageTags>
+    <Description>HealthChecks.Publisher.Datadog is the health check publisher for Datadog.</Description>
+    <Version>$(HealthCheckPublisherDatadog)</Version>
+    <RepositoryUrl>$(RepositoryUrl)</RepositoryUrl>
+    <Company>$(Company)</Company>
+    <Authors>$(Authors)</Authors>
+    <PackageId>AspNetCore.HealthChecks.Publisher.Datadog</PackageId>
+    <PublishRepositoryUrl>$(PublishRepositoryUrl)</PublishRepositoryUrl>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder)</AllowedOutputExtensionsInPackageBuildOutputFolder>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="DogStatsD-CSharp-Client" Version="3.3.0" />
```

Comment:
Does this dependency need to be added to dependencies.json? Not too familiar with that file and not sure what uses it.
@unaizorrilla, @lurumad, @carloslanderas, @eiximenis

Reply:
Yeap, please add this version ( DotStatsD-csharp-client) also in dependencies.props!